### ISSUE TITLE
Reproduce issue

### ISSUE THREAD

**Issue by jinxxo-j** [gmberton/EigenPlaces#6]
Hello! 
I'm very interested about Eigenplaces and want to reproduce the recall. 
However, I'm getting the error I've attached below.
How can I resolve it?

request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620865.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620866.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620867.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620868.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620869.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620870.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620871.png
request failed [400]: https://a.tile.openstreetmap.org/22/670283/1620872.png

**Comment by gmberton** [comment]
Hi, can you send the content of the file `info.log` that is generated during training?

**Comment by jinxxo-j** [comment]
Hello.

info.log file is here:

2024-01-11 06:04:00   train.py --train_dataset_folder /mnt/hdd2/jinwoo/sf_xl/raw/train/panoramas --val_dataset_folder /mnt/hdd2/jinwoo/sf_xl/processed/val --test_dataset_folder /mnt/hdd2/jinwoo/sf_xl/processed/test --visualize_classes 3
2024-01-11 06:04:00   Arguments: Namespace(M=15, N=3, focal_dist=10, s=100, m=0.4, lambda_lat=1.0, lambda_front=1.0, groups_num=9, min_images_per_class=5, backbone='ResNet50', fc_output_dim=2048, batch_size=32, epochs_num=40, iterations_per_epoch=5000, lr=1e-05, classifiers_lr=0.01, brightness=0.1, contrast=0.1, hue=0.1, saturation=0.1, random_resized_crop=0.5, infer_batch_size=16, positive_dist_threshold=25, resume_train=None, resume_model=None, device='cuda', seed=0, num_workers=8, visualize_classes=3, train_dataset_folder='/mnt/hdd2/jinwoo/sf_xl/raw/train/panoramas', val_dataset_folder='/mnt/hdd2/jinwoo/sf_xl/processed/val', test_dataset_folder='/mnt/hdd2/jinwoo/sf_xl/processed/test', save_dir='default')
2024-01-11 06:04:00   The outputs are being saved in logs/default/2024-01-11_06-04-00
2024-01-11 06:04:00   Loading pretrained backbone's weights from CosPlace
2024-01-11 06:04:02   There are 2 GPUs and 48 CPUs.
2024-01-11 06:04:03   Using cached dataset cache/sfxl_M15_N3_mipc5.torch

**Comment by gmberton** [comment]
I don't see any errors in the file. Does the training stops?
The messages of "request failed [400]" might be due to [this line](https://github.com/gmberton/EigenPlaces/blob/main/datasets/eigenplaces_dataset.py#L133), which is a part of the code used to generate maps of the classes, which can be useful for visualization purposes. Those visualizations are not needed for the training, so if it doesn't crash the training you can ignore those messages.
Hope this helps!

**Comment by jinxxo-j** [comment]
Thanks a lot for help !
